lib/cretonne/src/ir/entities.rs

```diff
@@ -23,7 +23,7 @@ use std::fmt;
 use std::u32;
 
 /// An opaque reference to an extended basic block in a function.
-#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug, PartialOrd, Ord)]
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug, PartialOrd, Ord, Default)]
```

Comment:
That was a corner case between `IlBuilder` initialization and the beginning of the translation for the `Position` struct, It is not used or read so I don't actually need a proper default value.